src/main/scala/emm/effects/Binder.scala

```diff
@@ -29,6 +30,22 @@ trait BinderLowPriorityImplicits {
       NAP.pack(back)
     }
   }
+
+  implicit def binderState[S, C <: Effects](implicit C: Binder[C], T: Traverser[C], NN: NonNested[C]): Binder[State[S, ?] |: C] = new Binder[State[S, ?] |: C] {
```

Comment:
That is a very good question. As I think about it, is supporting `State` at all even the right thing to do? Someone can always throw `Eval` or whatever uneffectful thing in for `F[_]` in `StateT` to achieve `State`. So do we need `State` at all, except for convenience?

On the flip side, the fact that we may *need* to support some monad transformer `WhateverT` has me questioning if `Emm` is general enough. The problem with `State` is the inner `F` in `F[S => F[(S, A)]]`. We're not *just* wrapping `State` in some monad. We're also wrapping the return of the function, namely `F[(S, A)]`. This turns it into a special case, which I don't like, because I don't like special cases...

I think it's worth some real though about how to avoid these special cases.